win32ss/gdi/gdi32/objects/bitmap.c

```diff
@@ -767,14 +742,12 @@ SetDIBitsToDevice(
      pConvertedInfo->bmiHeader.biCompression  == BI_PNG )) )*/
     {
         LinesCopied = NtGdiSetDIBitsToDeviceInternal(hdc, XDest, YDest, Width, Height, XSrc, YSrc,
-            StartScan, ScanLines, (LPBYTE) pvSafeBits, (LPBITMAPINFO) pConvertedInfo, ColorUse,
+            StartScan, ScanLines, (LPBYTE) Bits, (LPBITMAPINFO) pConvertedInfo, ColorUse,
```

Comment:
`NtGdiSetDIBitsToDeviceInternal->GreCreateBitmapEx->DecompressBitmap` does access this under SEH,
but it never propagates an error so it will continue on uninitialized data afterwards, is that expected behavior?

Reply:
Uh, yeah that whole code path seems off. If the bitmap is not compressed, the UM pointer is directly saved to the surface object, it seems; if they're compressed then we go through this code path, but I think we go through this one first:  https://github.com/reactos/reactos/blob/master/win32ss/gdi/eng/surface.c#L265-L275 -- which means the destination buffer is zero-initialized. So at least that `DecompressBitmap` aspect should be safe (no idea whether it's correct, though).